src/Lexer.php

```diff
@@ -108,9 +116,21 @@ public function analyse($input, $language = 'en')
         $this->allowMultilineArguments = false;
         $this->allowSteps = false;
 
-        $this->setLanguage($language);
+        if (\func_num_args() > 1) {
+            // @codeCoverageIgnoreStart
+            \assert($language !== '');
+            // TODO trigger deprecation (the Parser does not use this code path)
+            $this->setLanguage($language);
+        // @codeCoverageIgnoreEnd
```

Comment:
Minor formatting issue, should be indented?

Reply:
php-cs-fixer made me un-indent it, because it considers that a comment just before the `else` is related to the `else` clause (comments are generally put before the thing they explain)